iOS_SDK/OneSignalSDK/Source/OneSignal.h

```diff
@@ -177,6 +177,30 @@ typedef NS_ENUM(NSUInteger, OSNotificationDisplayType) {
 
 @end
 
+/* OneSignal OSNotificationGenerationJob used in notificationWillShowInForegroundHandler. The display type for the notification can be changed before it is presented.*/
+@interface OSNotificationGenerationJob : NSObject
+
+/* Display method of the notification */
+@property(nonatomic) OSNotificationDisplayType displayType;
```

Comment:
can you remove space, in order to follow same style with the other properties 😀

Reply:
fixed!